plane/source/docs/qautotune-mode.rst

```diff
@@ -27,4 +27,6 @@ You can reposition while tuning by moving your sticks. Once you re-center the st
 
 You can read more about AutoTune for copters here: :ref:`autotune` . The autotune for quadplanes works the same way.
 
+.. note:: Like Copter, entering QAUTOTUNE from QLOITER will attempt a loose position hold. However, this does not apply for yaw tunes and the vehicle will drift with the wind during yaw axis tuning.
```

Comment:
it does apply to yaw

Reply:
yes, for copter as well it will hold position during yaw tunes

Reply:
I must have imagined it....I did not let it drift long, when I thought it got outside the hold limits I stopped the flight......will fix this....sorry for the red herring